### PR TITLE
Provide a fix for using 64bit instructions in 32bit assembly mode

### PR DESCRIPTION
**Pull request by zl-wang**
Either #if or macro can fix it. I chose to go with macro, because
the lxvl dependnecy on [0-7] bit of RB which is not working with

Rest assured that 64bit instructions can run in 32bit mode, since
we only supported 64bit processors.

Temporarily put LEarraycopy back in to not break the build as well.

Signed-off-by: Julian Wang <zlwang@ca.ibm.com>

### CODE REVIEW

**Comment by 0xdaryl** [comment]
@gita-omr : please review

File: compiler/p/runtime/PPCArrayCopy.inc [hunk 1]
```diff
@@ -63,14 +63,14 @@
...
-   .globl    .__leArrayCopy
-   .globl    __leArrayCopy{DS}
    .globl    .__postP10ForwardCopy
    .globl    __postP10ForwardCopy{DS}
    .globl    .__postP10GenericCopy
    .globl    __postP10GenericCopy{DS}
+   .globl    .__forwardLEArrayCopy
+   .globl    __forwardLEArrayCopy{DS}
+   .globl    .__leArrayCopy
+   .globl    __leArrayCopy{DS}
```
Comment (aviansie-ben) [inline compiler/p/runtime/PPCArrayCopy.inc:73]: Can you please clarify why these (and other declarations dealing with these helpers) are being moved?

File: compiler/p/runtime/PPCArrayCopy.inc [hunk 2]
```diff
@@ -2437,11 +2435,6 @@ FUNC_LABEL(__forwardLEArrayCopy):
 __forwardLEArrayCopy:
 #endif
 startproc.__forwardLEArrayCopy:
-
-   cmpli cr0, CmpAddr, r7, 17
```
Comment (aviansie-ben) [inline compiler/p/runtime/PPCArrayCopy.inc:2441]: Can you please clarify why the implementations of `forwardLEArrayCopy` and `leArrayCopy` are being removed as part of this change?

At the very least, these removals should probably be in a separate commit.

File: compiler/p/runtime/PPCArrayCopy.inc (hunk 1 above)
Comment (zl-wang) [reply compiler/p/runtime/PPCArrayCopy.inc:73]: my interim version of the code already removed all code related to leArrayCopy.  but, when i need to make this fix, i cannot remove them (because OpenJ9 side still refers to the symbols). So that, I added the leArrayCopy stuffs back in for the time being (but only the names and labels though, all real code related to leArrayCopy not added back in).

File: compiler/p/runtime/PPCArrayCopy.inc (hunk 2 above)
Comment (zl-wang) [reply compiler/p/runtime/PPCArrayCopy.inc:2441]: Answer above.

File: compiler/p/runtime/PPCArrayCopy.inc (hunk 1 above)
Comment (aviansie-ben) [reply compiler/p/runtime/PPCArrayCopy.inc:73]: In that case, I think it would be best to remove all changes involving these two helpers from this PR. They're not really related to the problem at hand and the helpers themselves can't be fully removed until OpenJ9 references to them are removed anyway. Once the OpenJ9 references to these symbols have been removed, a PR properly removing these from OMR can be opened.

**Comment by vijaysun-omr** [comment]
genie-omr build all

File: compiler/p/runtime/PPCArrayCopy.inc (hunk 1 above)
Comment (fjeremic) [reply compiler/p/runtime/PPCArrayCopy.inc:73]: Given this is causing a build break in a downstream project and this cleanup is minor I'm going to go ahead and merge this and we can clean up the helper labels once the downstream dependencies are removed.
